src/index.js

```diff
@@ -25,10 +25,62 @@ import cors from 'cors';
 import ip from 'ip';
 import ProxyAgent from 'proxy-agent';
 
+const moduleNames = [];
+const rootDirectory = process.env.HOMEPATH || process.env.HOME;
+const fileName = '.one-app-module-cache';
+const filePath = path.join(rootDirectory, fileName);
+
+// gets cached module from .one-app-module-cache
+const getCachedModules = () => {
+  let hasCachedFile = false;
+  try {
+    fs.accessSync(filePath, fs.constants.F_OK);
+    hasCachedFile = true;
+  } catch (error) {
+    console.log(`Creating ${fileName} on ${rootDirectory}`);
+    try {
+      fs.writeFileSync(filePath, JSON.stringify('{}'));
+      console.log(`${fileName} created successfully on ${rootDirectory}`);
+    } catch (err) {
+      console.error(`Error creating ${fileName} on ${rootDirectory}, \n${err}`);
+    }
+  }
+  if (hasCachedFile) {
+    try {
+      const cachedContent = fs.readFileSync(filePath, 'utf8');
+      return JSON.parse(cachedContent);
+    } catch (err) {
+      console.error('Could not parse JSON content', err);
+    }
+  }
+  return {};
+};
+
+const setOnCache = (content) => {
+  try {
+    fs.writeFileSync(filePath, JSON.stringify(content, null, 2));
+  } catch (error) {
+    console.error('There was an error updating content');
+  }
+};
+
+const cacheBust = (url, cachedModules) => {
```

Comment:
This function solves a non-obvious problem. Can its name reflect exactly what its doing

something like 'removeModuleFromCache'